<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quick_clientClass</class>
 <widget class="QMainWindow" name="quick_clientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>quick_client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>534</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>本地端口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_port">
           <property name="text">
            <string>6060</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_start">
           <property name="text">
            <string>启动</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>连接1</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>目标ip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_dst_ip">
              <property name="text">
               <string>192.168.1.195</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>目标端口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_dst_port">
              <property name="text">
               <string>8088</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_connect">
              <property name="text">
               <string>连接</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_disconnect">
              <property name="text">
               <string> 断开</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>消息</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_msg">
              <property name="text">
               <string>hello quick_server</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_send">
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>消息</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_msg_loop">
              <property name="text">
               <string>hello quickkkkkkkkkkkkkkkkkkkk</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>发送间隔ms</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_send_interval">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btn_send_loop">
            <property name="text">
             <string>定时发送</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>总发送字节：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_total_send">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>总接收字节：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_total_recv">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>连接2</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>目标ip</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_dst_ip_2">
              <property name="text">
               <string>192.168.1.195</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>目标端口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_dst_port_2">
              <property name="text">
               <string>8088</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_connect_2">
              <property name="text">
               <string>连接</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_disconnect_2">
              <property name="text">
               <string> 断开</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>消息</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_msg_2">
              <property name="text">
               <string>hello quick_server</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_send_2">
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>消息</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_msg_loop_2">
              <property name="text">
               <string>hello quickkkkkkkkkkkkkkkkkkkk</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>发送间隔ms</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_send_interval_2">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btn_send_loop_2">
            <property name="text">
             <string>定时发送</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>总发送字节：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_total_send_2">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>总接收字节：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_total_recv_2">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>263</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_export_debug_info">
         <property name="text">
          <string>导出调试信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clear">
         <property name="text">
          <string>清除信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="quick_client.qrc"/>
 </resources>
 <connections/>
</ui>
